The error is completely unrelated:
```
2026-04-14 23:20:25,754 438 INFO odoo odoo.addons.account_analytic_document_date.tests.test_document_date: ====================================================================== 
2026-04-14 23:20:25,754 438 ERROR odoo odoo.addons.account_analytic_document_date.tests.test_document_date: FAIL: TestAccountMoveDocumentDate.test_create_invoice_with_document_date
Traceback (most recent call last):
  File "/__w/account-analytic/account-analytic/account_analytic_document_date/tests/test_document_date.py", line 95, in test_create_invoice_with_document_date
    self.assertNotEqual(invoice.invoice_date, invoice.analytic_document_date)
AssertionError: datetime.date(2026, 4, 15) == datetime.date(2026, 4, 15)
```

--- a/account_analytic_distribution_manual/static/src/components/analytic_distribution/analytic_distribution.esm.js
+++ b/account_analytic_distribution_manual/static/src/components/analytic_distribution/analytic_distribution.esm.js
@@ -43,12 +43,14 @@ patch(AnalyticDistribution.prototype, "account_analytic_distribution_manual", {
     },
     async save() {
         await this._super();
-        await this.props.record.update({
-            manual_distribution_id: [
-                this.state_manual_distribution.id,
-                this.state_manual_distribution.label,
-            ],
-        });
+        if (this.state_manual_distribution.id) {
+            await this.props.record.update({
+                manual_distribution_id: [
+                    this.state_manual_distribution.id,
+                    this.state_manual_distribution.label,
+                ],
+            });
+        }
     },
     async refreshManualDistribution(manual_distribution_id) {
         if (manual_distribution_id === 0) {

